src/sentry/integrations/github/integration.py

```diff
@@ -650,11 +653,9 @@ class GitHubIntegrationProvider(IntegrationProvider):
 
     setup_dialog_config = {"width": 1030, "height": 1000}
 
-    def get_client(self) -> GitHubBaseClient:
-        # XXX: This is very awkward behaviour as we're not passing the client an Integration
-        # object it expects. Instead we're passing the Installation object and hoping the client
-        # doesn't try to invoke any bad fields/attributes on it.
-        return GitHubApiClient(integration=self.integration_cls)
+    @property
+    def client(self) -> GithubSetupApiClient:
+        return GithubSetupApiClient()
```

Comment:
can we have a comment that this uses jwt? i was confused that we're not passing access token here